### DISCUSSION TITLE
Numbers Disappear in NumberInput When Typing Taiwanese Bopomofo

### DISCUSSION THREAD

**Discussion by ted2yma** [chakra-ui/chakra-ui#8918] [General]
### Description

When I type Bopomofo into the NumberInput, the numbers disappear one by one. This only happens on Chromium browsers (Chrome, Edge, Brave) on Windows. It works fine on Firefox on Windows and Chromium browsers on macOS.

---- Chinese version below ----
當你在NumberInput中輸入注音符號時，本來在NumberInput中的數字會逐一消失。這個情況只會在Windows的Chromium內核瀏覽器上發生，Windows版Firefox和MacOS上的瀏覽器是正常運作的，沒有任何數字消失。
*PS: Chromium內核的瀏覽器我只測試了Chrome、Edge與Brave。

### Link to Reproduction

https://codesandbox.io/p/sandbox/loving-franklin-fm37jd

### Steps to reproduce

1. Enter numbers in NumberInput (with or without min-max range).
2. Switch to Bopomofo input method and activate Chinese IME mode by clicking '中' on the taskbar.
3. Type Bopomofo characters key by key (use different keys, not the same one repeatedly).
4. Watch the numbers disappear.

--- Chinese version below ---
1. 先輸入任何數字，不管是否設置min-max。
2. 切換至注音輸入法
3. 在NumberInput中輸入不同的注音符號
4. 你會看到數字逐一消失

### Chakra UI Version

latest

### Browser

Google Chrome 124

### Operating System

- [ ] macOS
- [X] Windows
- [ ] Linux

**Comment by segunadebayo** [comment]
Closing as stale. Please start a new discussion if this is still relevant on the latest version.
